tests/version.py

```diff
@@ -287,5 +298,31 @@ def write_ks_cfg(buf):
         self.assertRaises(KickstartVersionError, versionFromFile, ks_cfg)
         os.unlink(ks_cfg)
 
+class FailedImpImport_TestCase(CommandTest):
+    def runTest(self):
+        try:
+            import sys
+
+            # will force another import
+            del sys.modules['pykickstart.version']
+            del sys.modules['imp']
+
+            # will cause ImportError
+            sys.modules['imputil'] = None
+
+            import pykickstart.version as ver
+            from pykickstart.handlers.f23 import F23Handler
+
+            cls = ver.returnClassForVersion(ver.F23)
+
+            # assert the names; b/c of how the imp.load_module() works
+            # asserting both classes being equal fails
+            self.assertEqual(cls.__name__, F23Handler.__name__)
+        finally:
+            # force import to reload these modules
+            del sys.modules['pykickstart.version']
+            del sys.modules['imp']
+            del sys.modules['imputil']
```

Comment:
I don't think this is necessary. In my test executions this test case is running somewhere in the middle and there are others (from the same module) running after it without any issues.